examples/stocks.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { LockAndCache } from "../lib";
+
+const cache = new LockAndCache();
+
+// Standalone mode.
+function getStockQuote(symbol: string) {
+  return cache.get(["stock", symbol], { ttl: 60 }, async () => {
+    // Fetch stock price from remote source, cache for one minute.
+    //
+    // Calling this multiple times in parallel will only run it once.
+    return 100;
+  });
+}
+
+// Wrap mode.
+async function stockQuoteHelper(symbol: string) {
+  // Fetch stock price from remote source, cache for one minute.
+  //
+  // Calling this multiple times in parallel will only run it once the cache key
+  // is based on the function name and arguments.
+  return 100;
+}
+const stockQuote = cache.wrap({ ttl: 60 }, stockQuoteHelper);
```

Comment:
again, this function is the same, you can just use the first function to demonstrate wrapping

Reply:
No, I don't want to demonstrate a double-wrapped function.

Both `wrap` and `get` need to be demonstrated here, partly because I want to make sure that TypeScript is actually happy with more complicated examples. :-)